pylint_django/tests/conftest.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This variable and the import above don't seem to be needed.

Reply:
Oh good catch. This was left over from before when I was checking to see if Django was installed. Will fix.

Reply:
Good point. Don't think `DEBUG` is needed either.